resources/public/api/conf/1.0/application.yaml

```diff
@@ -2,48 +2,822 @@ openapi: "3.0.3"
 
 info:
   version: 1.0.0
-  title: Hello World
+  title: DISA Returns API
 
 servers:
-  - url: https://test-api.service.hmrc.gov.uk/hello-world
+  - url: https://test-api.service.hmrc.gov.uk/disa-returns
     description: Sandbox
-  - url: https://api.service.hmrc.gov.uk/hello-world
+  - url: https://api.service.hmrc.gov.uk/disa-returns
     description: Production
 
-components:
-  parameters:
-    acceptHeader:
-      name: Accept
-      in: header
-      schema:
-        type: string
-        enum: [
-          "application/vnd.hmrc.1.0+json",
-          "application/vnd.hmrc.1.0+xml"
-        ]
-      required: true
-
 paths:
-  /hello-world:
+  /monthly-returns/{isaManagerReferenceNumber}/init:
+    post:
+      summary: Initialise a bulk monthly returns submission.
+      description: |
+        An endpoint that checks if submission is allowed and returns return ID.
+      tags:
+        - Monthly Returns Submission
+      security:
+        - userRestricted:
+            - write:isa-returns
+      parameters:
+        - name: isaManagerReferenceNumber
+          in: path
+          description: The reference given to the ISA provider when they applied for
+            approval from HMRC, commonly referred as Z-Ref.
+          required: true
+          style: simple
+          explode: false
+          schema:
+            pattern: "^Z([0-9]{4}|[0-9]{6})$"
+            type: string
+            example: Z1234
+        - $ref: '#/components/parameters/acceptHeader'
+        - $ref: '#/components/parameters/authorizationHeader'
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              type: object
+              required:
+                - totalRecords
+                - submissionPeriod
+                - taxYear
+              properties:
+                totalRecords:
+                  type: integer
+                  minimum: 0
+                  example: 10000
+                  description: 'Total number of records to be submitted'
+                submissionPeriod:
```

Comment:
Not sure I love this - @zachhall1234 do you know where the idea of using an ENUM rather than just 1-12 came from?